build.cake

```diff
@@ -50,7 +50,6 @@ Task("Restore-NuGet-Packages")
 	DotNetCoreRestore(new DotNetCoreRestoreSettings
 	{
 		NoCache = true,
-        ConfigFile = "./nuget.config",
```

Comment:
Please keep this

Reply:
This was a mistake due to merge from master.
Reverted.